### ISSUE TITLE
Move code to create orphan branch and push inside sandpaper

### ISSUE THREAD

**Issue by zkamvar** [carpentries/sandpaper#110]
At the moment, it's failing because I foolishly forgot to escape the full stop for grep: https://github.com/zkamvar/sandpaper-docs/runs/2385568562?check_suite_focus=true

```
 cd built/
  git config --local user.email "actions@github.com"
  git config --local user.name "GitHub Actions"
  CURR_HEAD=$(git rev-parse HEAD)
  git checkout --orphan md-outputs-PR-${NR}
  git add -A 
  git commit -m "source commit: ${CURR_HEAD}"
  cd ..
  ls -d -A built/* | grep -v '.git' | xargs rm -r
  unzip -d built built.zip
  cd built
  git add -A
  git commit --allow-empty -m "differences for PR #${NR}"
  git push -u --force --set-upstream origin md-outputs-PR-${NR}
  shell: /usr/bin/bash -e {0}
  env:
    NR: 8
Switched to a new branch 'md-outputs-PR-8'
[md-outputs-PR-8 (root-commit) 9610298] source commit: 60883913228ab8e12553f015afc78fe7348e9eac
 19 files changed, 1757 insertions(+)
 create mode 100644 CODE_OF_CONDUCT.md
 create mode 100644 LICENSE.md
 create mode 100644 deployment.md
 create mode 100644 editing.md
 create mode 100644 episodes.md
 create mode 100644 example.md
 create mode 100644 fig/carpentries-hex-blue.svg
 create mode 100644 fig/episodes-rendered-pyramid-1.png
 create mode 100644 fig/example-rendered-pyramid-1.png
 create mode 100644 fig/github-pages-gh-pages.png
 create mode 100644 fig/github-pages-none.png
 create mode 100644 github-pat.md
 create mode 100644 index.md
 create mode 100644 instructor.md
 create mode 100644 introduction.md
 create mode 100644 maintainer.md
 create mode 100644 md5sum.txt
 create mode 100644 notes.md
 create mode 100644 setup.md
Archive:  built.zip
  inflating: built/CODE_OF_CONDUCT.md  
  inflating: built/deployment.md     
  inflating: built/editing.md        
  inflating: built/episodes.md       
  inflating: built/example.md        
   creating: built/fig/
replace built/github-pat.md? [y]es, [n]o, [A]ll, [N]one, [r]ename:  NULL
(EOF or read error, treating as "[N]one" ...)
  inflating: built/index.md          
  inflating: built/instructor.md     
  inflating: built/introduction.md   
  inflating: built/LICENSE.md        
  inflating: built/maintainer.md     
  inflating: built/md5sum.txt        
  inflating: built/notes.md          
  inflating: built/setup.md          
  inflating: built/fig/carpentries-hex-blue.svg  
  inflating: built/fig/episodes-rendered-pyramid-1.png  
  inflating: built/fig/example-rendered-pyramid-1.png  
  inflating: built/fig/github-pages-gh-pages.png  
  inflating: built/fig/github-pages-none.png  
```

**Comment by zkamvar** [comment]
Found in https://github.com/zkamvar/sandpaper-docs/pull/8

**Comment by zkamvar** [comment]
This was partially addressed in https://github.com/zkamvar/actions/commit/bc7883c867e1eed065027938e44a242f1257a196, but it still needs a less fragile solution.

**Comment by zkamvar** [comment]
I believe this was addressed a while ago and formally tested in #137
